### PR TITLE
SOM reloaded

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
This PR is going to be the first of many to refresh the SOM into a genuine opposing force for the TGMC, starting by updating their hostile ERT and giving them some suitably mean weaponry.

image

New armour:
image

With Snazzy new armour sprites from Kuro, they are roughly equivalent to TGMC armour (being modular armour, they are fully compatible with the various armour attachments), except with slightly better bullet armour but worse laser armour. 
They also have better armour against the more uncommon damage types like energy, fire etc (mainly because they use all that shit, and getting fried by your own guns isn't fun). 
The Gorgon armor for the SOM leader is a small step above the heavy armour equivalent, and in the ERT at least, comes with Valk to be extra stubborn.

New weapons:
image

The SOM have unleashed their volkite weaponry, Martian ray guns that hit softer energy armour to fry people from the inside.
Every hit on a mob has a chance to deflagrate the target, causing them (and anyone around them) to burst into flames, inflicting extra damage and some brief debuffs.
The odds of this scale with the amount of damage done to the target. This takes into account their armour, so targets with little to no armour are significantly more likely to deflagrate and die horribly.

Due to the manufacture of these weapons, their attachment options are extremely limited - significantly, no underbarrel weapons. In addition, because they're energy weapons, they don't break bones/cause shrap and all that fun stuff. So while they're likely better DPS than just blasting away with a normal rifle, they do give up quite a lot for it.

To begin with, I've added 3 volkite weapons.

Volkite charger:
The smallest of the group, with low speed penalties, fast wield time and rate of fire. Roughly equivalent to an SMG or carbine in performance, and is not effective at longer ranges. Can be wielded effectively one handed, especially with a gyro.
Takes volkite energy cells, for 45 shots per mag.

https://user-images.githubusercontent.com/7869430/165416154-54d0e362-4119-481f-b6ed-71da96e55121.mp4

Volkite Caliver:
The main rifle, with good damage and capacity, and handling stats a little worse than a T12. Effective at all ranges.
Takes high capacity volkite energy cells, for 50 shots per mag (they use notably more charge per shot). Alternatively, they can also be linked to the powerback for considerably more capacity, however this will slow you down more.

https://user-images.githubusercontent.com/7869430/165416167-abbd7db1-bff9-49b8-aed0-aaf58763853e.mp4

Volkite culverin
The big boy, requires the back mounted powerpack to operate. Has damage per shot between the charger and caliver, but excellent rate of fire. For when you need to absolutely hose down a target. Has very poor mobility however.
Has 160 shots base with the powerpack. You will burn through this extremely fast however, and the recharge WILL NOT keep up. Don't go nuts.

https://user-images.githubusercontent.com/7869430/165416176-9027f1ce-8544-4565-895a-e8aebeb36d2a.mp4

The powerpack as well as providing a large amount of charge, (slowly) recharges itself over time, and provides some team utility by being able to charge cells, in the same way medics recharge defibs.

New ERT loadouts:
The ERT has been refreshed, with every role having at least 2 different load outs. The main difference are the weapons so as a brief summary:

Squad leader:
1. Charger with magharness + gyro
2. Caliver with magharnes + lasersight
3. Caliver with tac sensor +  laser sight (with powerpack and belt harness)

Veteran:
1. Charger with magharness + gyro
2. Caliver with magharnes + lasersight
3. Caliver with magharness +  laser sight (with powerpack and sawn off on the belt)
4. Culverin with magharness (with powerpack and sawn off on the belt)

Medic:
1. Charger with magharness
2. AK with magharness and bayo (hey, they can't all have the fancy stuff)

Standard:
1. Charger with magharness
2. AK with magharness and bayo

Veterans are the main bulk of the squad, and should form 70% of the basic dudes, with the standards filling the rest.
This should give some variety to each ERT, but you could in theory have like 8 dudes with culverins which would be entertaining.

HvH specific SOM ERT has been removed, as these guys should be able to carry themselves as a hostile ERT or dedicated HvH force. I haven't removed all the old sprites and stats from the code however.

Finally, in addition to a bunch of minor fixes I can't be bothered listing (and because I've probably forgot most of them), I have tweaked marine energy armour so it scales with light > medium > heavy (currently it was just flat).

There will be many more updates to come hopefully, starting with basic stuff like SOM specific armour attachments (as even if they're functionally the same as TGMC gear, they would be superficially different), but this is the minimum viable product to get them off the ground and see how people like them.

## Why It's Good For The Game
Update ERT to make marines sad
Make SOM great, paving the way for future content

## Changelog
:cl:
add: Volkite weaponry, Martian ray guns for SOM use only
expansion: Updates SOM ERT with new weaponry, armour and loadouts. You better be sure before you hit that distress beacon.
balance: Marine armour energy armour scales with weight class
fix: fixed a few misc sprite issues
/:cl:

### CODE REVIEW

**Comment by Arctrooper8** [comment]
B-But my insurgent miner lore.

**Comment by RipGrayson** [comment]
As sad as I am to see it go, this removal is needed. RIP my favorite piece of exploitable engineering gear.

File: code/modules/clothing/modular_armor/modular.dm [hunk 1]
```diff
@@ -789,4 +790,143 @@
...
+	icon_state_variants = list(
+		"black",
+	)
+	current_variant = "black"
+
+	allowed_uniform_type = /obj/item/clothing/under
+
+/obj/item/clothing/suit/modular/som/light
+	name = "\improper SOM scout armor"
+	desc = "The M-11 scout armor is A lightweight suit that that allows for minimal encumberance while still providing reason protection. Often seen on scouts or other specialist units that aren't normally getting shot at. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:835]:
```suggestion
	desc = "The M-11 scout armor is a lightweight suit that that allows for minimal encumberance while still providing reasonable protection. Often seen on scouts or other specialist units that aren't normally getting shot at. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 2]
```diff
@@ -789,4 +790,143 @@
...
+	soft_armor = list("melee" = 40, "bullet" = 65, "laser" = 55, "energy" = 55, "bomb" = 50, "bio" = 50, "rad" = 50, "fire" = 50, "acid" = 45)
+	icon_state = "som_light"
+	item_state = "som_light"
+	slowdown = 0.3
+
+/obj/item/clothing/suit/modular/som/heavy
+	name = "\improper SOM heavy battle armor"
+	desc = "A Standard suit of M-31 heavy duty combat armor worn by SOM shock troops. Provides excellent protection however it does reduce mobility somewhat. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:843]:
```suggestion
	desc = "A standard suit of M-31 heavy duty combat armor worn by SOM shock troops. Provides excellent protection however it does reduce mobility somewhat. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 3]
```diff
@@ -789,4 +790,143 @@
...
 	name = "\improper M11X pattern leader helmet"
 	desc = "A slightly fancier helmet for marine leaders. This one has cushioning to project your fragile brain."
-	soft_armor = list("melee" = 75, "bullet" = 65, "laser" = 50, "energy" = 50, "bomb" = 50, "bio" = 50, "rad" = 50, "fire" = 50, "acid" = 50)
+	soft_armor = list("melee" = 75, "bullet" = 65, "laser" = 55, "energy" = 55, "bomb" = 50, "bio" = 50, "rad" = 50, "fire" = 50, "acid" = 50)
+
+//SOM modular armour
+
+/obj/item/clothing/suit/modular/som
+	name = "\improper SOM light battle armor"
+	desc = "The M-21 battle armor is typically used by SOM light infantry, or other specialists that require more mobility at the cost of some protection. Provides good protection without minor impairment to the users mobility. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:799]:
```suggestion
	desc = "The M-21 battle armor is typically used by SOM light infantry, or other specialists that require more protection at the cost of some mobility. A standard combat battle armor for regular SOM tropos. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 4]
```diff
@@ -789,4 +790,143 @@
...
+
+/obj/item/clothing/head/modular/som/medic
+	starting_attachments = list(/obj/item/armor_module/storage/helmet/som_vet)
+
+/obj/item/clothing/head/modular/som/standard
+	starting_attachments = list(/obj/item/armor_module/storage/helmet/som)
+
+/obj/item/clothing/head/modular/som/veteran
+	name = "\improper SOM veteran helmet"
+	desc = "The standard combat helmet worn by SOM combat troops. State of the art materials provides more protection for more valuable brains."
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:918]:
```suggestion
	desc = "The standard combat helmet worn by SOM combat specialists. State of the art materials provides more protection for more valuable brains."
```

File: code/modules/projectiles/guns/energy.dm [hunk 5]
```diff
@@ -799,3 +799,143 @@
...
+	starting_attachment_types = list(/obj/item/attachable/magnetic_harness)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/caliver/somvet
+	starting_attachment_types = list(/obj/item/attachable/magnetic_harness, /obj/item/attachable/lasersight)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/caliver/tacsensor
+	starting_attachment_types = list(/obj/item/attachable/motiondetector, /obj/item/attachable/lasersight)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/culverin
+	name = "\improper Volkite culverin"
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/energy.dm:915]:
```suggestion
	name = "\improper Volkite culverin"
```
Is Volkite supposed to be captizalized or not? It's very inconsistent.

File: code/modules/projectiles/magazines/energy.dm [hunk 6]
```diff
@@ -55,3 +56,71 @@
...
+	icon = 'icons/obj/items/ammo.dmi'
+	icon_state = "volkite_big"
+	maxcharge = 1800
+	w_class = WEIGHT_CLASS_NORMAL
+	icon_state_mini = "mag_cell"
+	charge_overlay = "volkite_big"
+
+/obj/item/cell/lasgun/volkite/powerpack
+	name = "\improper M-70 powerpack"
+	desc = "A heavy reinforced backpack with an array of ultradensity energy cells, linked to a miniature radioisotope thermoelectric generator for continuous power generation. Used to power the largest man portable volkite weaponry"
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/energy.dm:85]:
```suggestion
	desc = "A heavy reinforced backpack with an array of ultradensity energy cells, linked to a miniature radioisotope thermoelectric generator for continuous power generation. Used to power the largest man portable volkite weaponry."
```

File: code/modules/projectiles/magazines/energy.dm [hunk 7]
```diff
@@ -55,3 +56,71 @@
...
+	charge_delay = 2 SECONDS
+
+/obj/item/cell/lasgun/volkite/powerpack/proc/use_charge(mob/user, amount = 0, mention_charge = TRUE)
+	var/warning = ""
+	if(amount > charge)
+		playsound(src, 'sound/machines/buzz-two.ogg', 25, 1)
+		if(charge)
+			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, its battery only having enough power to partially recharge the cell for [charge] amount. "
+		else
+			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, as its battery is completely depleted of charge. "
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/energy.dm:106]:
```suggestion
			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, as its battery is completely depleted of charge."
```

File: code/modules/projectiles/magazines/energy.dm [hunk 8]
```diff
@@ -55,3 +56,71 @@
...
+	else
+		playsound(src, 'sound/machines/ping.ogg', 25, 1)
+		warning = "<span class='notice'>[src]'s powerpack recharge unit cheerfully pings as it successfully recharges the cell. "
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/energy.dm:109]:
```suggestion
		warning = "<span class='notice'>[src]'s powerpack recharge unit cheerfully pings as it successfully recharges the cell."
```

File: code/modules/projectiles/magazines/energy.dm [hunk 9]
```diff
@@ -55,3 +56,71 @@
...
+	self_recharge = TRUE
+	charge_amount = 32
+	charge_delay = 2 SECONDS
+
+/obj/item/cell/lasgun/volkite/powerpack/proc/use_charge(mob/user, amount = 0, mention_charge = TRUE)
+	var/warning = ""
+	if(amount > charge)
+		playsound(src, 'sound/machines/buzz-two.ogg', 25, 1)
+		if(charge)
+			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, its battery only having enough power to partially recharge the cell for [charge] amount. "
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/energy.dm:104]:
```suggestion
			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, its battery only having enough power to partially recharge the cell for [charge] amount."
```

File: code/modules/projectiles/guns/energy.dm (hunk 5 above)
Comment (Lumipharon) [reply code/modules/projectiles/guns/energy.dm:915]: lowercase, it's just from me changing things/adding words and forgetting to decapitalise what was originally at the start of the sentence. Or not being clear on whether item names actually NEED capitalisation (looking now I think the answer is no)

File: code/modules/clothing/modular_armor/modular.dm (hunk 3 above)
Comment (Lumipharon) [reply code/modules/clothing/modular_armor/modular.dm:799]: Nah, the heavy armour is the "standard", to compare to. SOM lads are build to fight marines. I still have a gorillion typos and stuff though

**Comment by Lumipharon** [comment]
I swear I am actually dyslexic or something, christ.
Thanks for reading through all this.

**Comment by novaepee** [comment]
> I swear I am actually dyslexic or something, christ. Thanks for reading through all this.

XSlayer3000 humbles us all.

**Comment by Al-1ce** [comment]
Helmets look a little bit silly, and possibly respites of the EOD.

File: code/modules/clothing/modular_armor/attachments/uniform.dm [hunk 10]
```diff
@@ -161,6 +186,9 @@
 	. = ..()
 	new /obj/item/weapon/gun/pistol/vp70(storage)
 
+/obj/item/armor_module/storage/uniform/holster/highpower/Initialize()
+	. = ..()
+	new /obj/item/weapon/gun/pistol/highpower(storage)
 /obj/item/storage/internal/holster
```
Comment (TiviPlus) [inline code/modules/clothing/modular_armor/attachments/uniform.dm:192]:
```suggestion
	new /obj/item/weapon/gun/pistol/highpower(storage)

/obj/item/storage/internal/holster
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 11]
```diff
@@ -789,4 +790,143 @@
...
+		/obj/item/armor_module/module/fire_proof,
+		/obj/item/armor_module/module/tyr_extra_armor,
+		/obj/item/armor_module/module/tyr_extra_armor/mark1,
+		/obj/item/armor_module/module/mimir_environment_protection,
+		/obj/item/armor_module/module/mimir_environment_protection/mark1,
+		/obj/item/armor_module/module/hlin_explosive_armor,
+		/obj/item/armor_module/module/ballistic_armor,
+		/obj/item/armor_module/module/chemsystem,
+
+		/obj/item/armor_module/storage/general,
```
Comment (TiviPlus) [inline code/modules/clothing/modular_armor/modular.dm:817]:
```suggestion
		/obj/item/armor_module/module/chemsystem,
		/obj/item/armor_module/storage/general,
```

File: code/modules/clothing/shoes/marine_shoes.dm [hunk 12]
```diff
@@ -146,6 +146,10 @@
 	item_state = "som"
 	soft_armor = list("melee" = 30, "bullet" = 20, "laser" = 20, "energy" = 20, "bomb" = 30, "bio" = 20, "rad" = 20, "fire" = 20, "acid" = 25)
 
+/obj/item/clothing/shoes/marine/som/knife/Initialize()
+	.=..()
+	new /obj/item/weapon/combat_knife(pockets)
```
Comment (TiviPlus) [inline code/modules/clothing/shoes/marine_shoes.dm:151]:
```suggestion
	. = ..()
	new /obj/item/weapon/combat_knife(pockets)
```

File: code/modules/projectiles/ammo_datums.dm [hunk 13]
```diff
@@ -172,6 +176,40 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 		var/armor_block = victim.run_armor_check(null, proj.ammo.armor_type)
 		victim.apply_damage(proj.damage * proj.airburst_multiplier, proj.ammo.damage_type, null, armor_block, updating_health = TRUE)
 
+/datum/ammo/proc/deflagrate(atom/target, obj/projectile/proj)
+	if(!target || !proj)
+		CRASH("airburst() error: target [isnull(target) ? "null" : target] | proj [isnull(proj) ? "null" : proj]")
```
Comment (TiviPlus) [inline code/modules/projectiles/ammo_datums.dm:181]: copyapaste

File: code/modules/projectiles/ammo_datums.dm [hunk 14]
```diff
@@ -172,6 +176,40 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
...
+	var/mob/living/victim = target
+	var/armor_block = victim.run_armor_check(null, "fire") //checks fire armour across the victim's whole body
+	var/deflagrate_chance = (proj.damage * deflagrate_multiplier * (100 + min(0, proj.penetration - armor_block)) / 100)
+	if(prob(deflagrate_chance))
+		playsound(target, 'sound/effects/incendiary_explode.ogg', 45, falloff = 5)
+		fire_burst(target, proj)
+
+///the actual fireblast triggered by deflagrate
+/datum/ammo/proc/fire_burst(atom/target, obj/projectile/proj)
+	if(!target || !proj)
```
Comment (TiviPlus) [inline code/modules/projectiles/ammo_datums.dm:193]: why isnt this inlined

File: code/modules/projectiles/ammo_datums.dm [hunk 15]
```diff
@@ -172,6 +176,40 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
...
+		CRASH("airburst() error: target [isnull(target) ? "null" : target] | proj [isnull(proj) ? "null" : proj]")
```
Comment (TiviPlus) [inline code/modules/projectiles/ammo_datums.dm:194]: copypasteeee

File: code/modules/projectiles/magazines/energy.dm [hunk 16]
```diff
@@ -55,3 +56,71 @@
...
+	flags_equip_slot = ITEM_SLOT_BACK
+	flags_magazine_features = MAGAZINE_REFUND_IN_CHAMBER|MAGAZINE_WORN
+	w_class = WEIGHT_CLASS_HUGE
+	slowdown = 0.3
+	maxcharge = 4800
+	self_recharge = TRUE
+	charge_amount = 32
+	charge_delay = 2 SECONDS
+
+/obj/item/cell/lasgun/volkite/powerpack/proc/use_charge(mob/user, amount = 0, mention_charge = TRUE)
```
Comment (TiviPlus) [inline code/modules/projectiles/magazines/energy.dm:99]: autodoc

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/magazines/energy.dm [hunk 17]
```diff
@@ -55,3 +56,71 @@
...
+		warning = "<span class='notice'>[src]'s powerpack recharge unit cheerfully pings as it successfully recharges the cell."
+	. = min(charge, amount)
+	charge -= .
+	if(mention_charge)
+		to_chat(user, span_notice("[warning]<b>Charge Remaining: [charge]/[maxcharge]</b>"))
+	update_icon()
+
+/obj/item/cell/lasgun/volkite/powerpack/MouseDrop_T(obj/item/W, mob/living/user) //Dragging the power cell onto the backpack will trigger its special functionality.
+	if(istype(W, /obj/item/cell))
+		var/obj/item/cell/D = W
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/modules/projectiles/magazines/energy.dm:118]: Updated for cells needing to be in hand. Range check doesn't seem to be required for the powerpack, as I tested and can't remotely recharge cells.

File: code/modules/projectiles/magazines/energy.dm [hunk 18]
```diff
@@ -55,3 +56,71 @@
...
+		var/charge_difference = D.maxcharge - D.charge
+		if(charge_difference) //If the cell has less than max charge, recharge it.
+			var/charge_used = use_charge(user, charge_difference) //consume an appropriate amount of charge
+			D.charge += charge_used //Recharge the cell battery with the lower of the difference between its present and max cap, or the remaining charge
+			D.update_icon()
+		else
+			to_chat(user, span_warning("This cell is already at maximum charge!"))
+	return ..()
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/modules/projectiles/magazines/energy.dm:126]: done

File: code/modules/projectiles/magazines/energy.dm (hunk 16 above)
Comment (Lumipharon) [reply code/modules/projectiles/magazines/energy.dm:99]: fixed

File: code/modules/projectiles/ammo_datums.dm (hunk 15 above)
Comment (Lumipharon) [reply code/modules/projectiles/ammo_datums.dm:194]: fixed

File: code/modules/projectiles/ammo_datums.dm (hunk 14 above)
Comment (Lumipharon) [reply code/modules/projectiles/ammo_datums.dm:193]: OK'd

File: code/modules/projectiles/ammo_datums.dm (hunk 13 above)
Comment (Lumipharon) [reply code/modules/projectiles/ammo_datums.dm:181]: fixed

File: code/modules/projectiles/ammo_datums.dm [hunk 19]
```diff
@@ -172,6 +176,40 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 		var/armor_block = victim.run_armor_check(null, proj.ammo.armor_type)
 		victim.apply_damage(proj.damage * proj.airburst_multiplier, proj.ammo.damage_type, null, armor_block, updating_health = TRUE)
 
+/datum/ammo/proc/deflagrate(atom/target, obj/projectile/proj)
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/modules/projectiles/ammo_datums.dm:179]: fixed

File: code/modules/projectiles/magazines/energy.dm [hunk 20]
```diff
@@ -55,3 +56,71 @@
...
+	self_recharge = TRUE
+	charge_amount = 32
+	charge_delay = 2 SECONDS
+
+/obj/item/cell/lasgun/volkite/powerpack/proc/use_charge(mob/user, amount = 0, mention_charge = TRUE)
+	var/warning = ""
+	if(amount > charge)
+		playsound(src, 'sound/machines/buzz-two.ogg', 25, 1)
+		if(charge)
+			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, its battery only having enough power to partially recharge the cell for [charge] amount."
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/modules/projectiles/magazines/energy.dm:104]: Fixed, and no, minigun packs don't have this functionality. I assume.

File: code/modules/projectiles/magazines/energy.dm [hunk 21]
```diff
@@ -55,3 +56,71 @@
...
+		else
+			warning = "<span class='warning'>[src]'s powerpack recharge unit buzzes a warning, as its battery is completely depleted of charge."
+	else
+		playsound(src, 'sound/machines/ping.ogg', 25, 1)
+		warning = "<span class='notice'>[src]'s powerpack recharge unit cheerfully pings as it successfully recharges the cell."
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/modules/projectiles/magazines/energy.dm:109]: fixed

File: code/modules/projectiles/guns/energy.dm [hunk 22]
```diff
@@ -799,3 +799,143 @@
...
+
+	accuracy_mult = 1
+	scatter = 0
+	recoil = 0
+	accuracy_mult_unwielded = 0.5
+	scatter_unwielded = 25
+	recoil_unwielded = 3
+
+	aim_slowdown = 0.25
+	wield_delay	= 0.4 SECONDS
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/energy.dm:832]:
```suggestion
	wield_delay = 0.4 SECONDS
```

**Review by TiviPlus — APPROVED** [review]
Rest barring disc comment about armor ok

**Pull request merged**